resources/td-agent/rpm/rpmmacros.erb

```diff
@@ -0,0 +1,4 @@
+%_signature gpg
+%_gpg_path <%= gpg_path %>
+%_gpg_name <%= gpg_name %>
+%define debug_package %{nil}
```

Comment:
Your initial commit (with `%define _build_id_links none`) should have been sufficient… Any reason it didn't work?